### PR TITLE
Update Building and Using An MLOPs Stack With ZenML.ipynb

### PR DESCRIPTION
**Pull request by Ben-Epstein**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
 # nba-ml-pipeline
 A NBA pipeline that predicts 3-pointers.
+
+## Setup
+```
+git clone https://github.com/zenml-io/nba-ml-pipeline
+cd nba-ml-pipeline
+pip install -r requirements.txt
+pip install -r requirements_kubeflow.txt
```
Comment (htahir1) [inline README.md:9]: We dont need to install these :), we do them via `zenml integration install`

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 zenml
-nba-api
\ No newline at end of file
+nba-api
+sklearn
```
Comment (htahir1) [inline requirements.txt:3]: We do this via `zenml integration install`. Lets take this out and we'll create a README update for it

**Review by htahir1 — CHANGES_REQUESTED** [review]
Thanks for the awesome session and PR!

**Comment by htahir1** [comment]
@AlexejPenner Let's try and incoorporate @Ben-Epstein 's PR too in your PR? Ben, we're doing a big overhaul of the repo to make it more usable so these would be included

**Comment by AlexejPenner** [comment]
@Ben-Epstein thanks, i'll merge this and incorporate it into the repo overhaul

**Pull request merged**